problems/numberOfDigits.js

```diff
@@ -7,8 +7,27 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let num = n
+    return Math.floor(num.toString().length)
```

Comment:
No need for Math.floor. That's only for rounding a decimal down. .length on a string will always be an integer.